packages/@okta/vuepress-site/docs/reference/api/oie-policies/index.md

```diff
@@ -0,0 +1,737 @@
+|-------------|------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------|
+| name        | String                                                                       | Human-readable name for the Rule, configurable during creation or updating.                        |
+| id          | String                                                                       | Unique identifier for this Rule (read-only)                                                        |
+| type        | String                                                                       | Type of the Rule. For Identifier Match Rule Objects, this needs to be `Okta:UnknownUser`.          |
+| priority    | Integer                                                                      | Used to determine which Rules take precedence.                                                     |
+| conditions  | Array                                                                        | No conditions are supported for this Rule type, so this must be an empty array.                    |
+| action      | String                                                                       | Either `ALLOW` or `DENY`. Controls whether the user is allowed to proceed.                         |
+| requirement | [Unknown User Rule Requirement Object](#unknow-user-rule-requirement-object) | Specifies whether to allow an unknown user to register and, if so, what User Type to use for them. |
+| status      | String                                                                       | `ACTIVE`  or  `INACTIVE`.                                                                          |
+| default     | Boolean                                                                      | `true` for the first instance of this Rule, which gets created by default.                         |
+
+#### Unknown User Rule Requirement Object
+
+| Property                                   | Type   | Description                                                                                                                                                            |
+|--------------------------------------------|--------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `noUserMatch.action`                       | String | `REGISTER` or `DENY` to determine whether unknown users should be allowed to register.                                                                                 |
+| `noUserMatch.registration.defaultUserType` | String | Valid ID of a User Type. Sets the User Type to enroll unknown users as, if you have allowed unknown users to register. Not required if `noUserMatch.action` is `DENY`. |
```

Comment:
`noUserMatch.registration` must be omitted if `noUserMatch.action` is `DENY`

Reply:
Corrected description of noUserMatch.registration.defaultUserType to say must be omitted noUserMatch.action is DENY.